aiida/plugins/entry_point.py

```diff
@@ -50,6 +50,28 @@ class EntryPointFormat(enum.Enum):
     MINIMAL = 3
 
 
+class EntryPointCache():
```

Comment:
I think this might be equivalent to adding the ``@functools.lru_cache(maxsize=None)`` decorator to the ``get_entry_points`` function.

Reply:
Good point @greschd , yes this would be a better way to do it

Reply:
Thanks, that seems to be much more elegant